packages/wiz/data_stream/audit/manifest.yml

```
@@ -0,0 +1,80 @@
+title: Collect Audit logs from Wiz.
+type: logs
+streams:
+  - input: cel
+    title: Audit logs
+    description: Collect Audit logs from Wiz.
+    template_path: input.yml.hbs
```

```suggestion
    template_path: cel.yml.hbs
```

Applies to other datastreams too